src/main/java/com/rst/bootcamp/model/Instructor.java

```diff
@@ -0,0 +1,89 @@
+package com.rst.bootcamp.model;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.OneToOne;
+
+@Entity
+public class Instructor {
+
+	@Id
+	@GeneratedValue
+	private int id;
+
+	private String instructorName;
+
+	private String adminName;
+
+	@OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER, optional = true)
```

Comment:
classic tree structure
can have multiple subordinates, change to  OneToMany

Reply:
optional = true is redundant 
not required if true

Reply:
Did the tree structure.